addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -844,7 +844,55 @@ options.registry.BackgroundVideo = options.Class.extend({
     },
 });
 
-options.registry.OptionsTab = options.Class.extend({
+options.registry.WebsiteLevelColor = options.Class.extend({
+    specialCheckAndReloadMethodsNames: options.Class.prototype.specialCheckAndReloadMethodsNames
+        .concat(['customizeWebsiteLayer2Color']),
+
+    /**
+     * @see this.selectClass for parameters
+     */
+    async customizeWebsiteLayer2Color(previewMode, widgetValue, params) {
+        if (previewMode) {
+            return;
+        }
```

Comment:
I looked into this, and here is what I understand as the reason why this was done this way:
- `customizeWebsiteLayer2Color` does not impact the DOM, therefore it never has a visible preview
- because it never has a preview, `data-no-preview` is not an option: setting it would have no effect
- it therefore gets called as if it had a preview, with the various `previewMode` values
- both `customizeWebsite*` calls inside the method actually contain the same `if` - so, this is just shortcut logic

Reply:
In general I like `if (previewMode) { return }` in the JS method when the method **cannot** be used in previewMode (which is the case here: making a RPC, reloading the assets, etc is not something we'll want ever). Using data-no-preview is however better when the method could be called in previewMode but we decided not to (somehow could reuse the option somewhere with previewMode enabled).